bin/markdown-locales

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node --use_strict
+/* eslint-disable no-console */
+
+require('babel-register');
+
+const gmd = require('gettext-markdown');
+const config = require('config');
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+
+const supportedLangs = config.get('langs');
+const localeDir = path.join(__dirname, '../src/amo/guides/locale');
+const templateDir = path.join(localeDir, '../templates/LC_MESSAGES/');
+const langToLocale = require('../src/core/i18n/utils').langToLocale;
+
+let lang;
+let locale;
+let outputFile;
+
+// Get guide md files.
+const files = fs.readdirSync(__dirname + '/../src/amo/guides/');
+const mdFiles = files.filter(file => path.extname(file) === '.md');
+
+for (file of mdFiles) {
+  // TODO: uncomment the following for loop when ready for all locales.
+  // For now we'll use locale af to sample this.
+  // for (supportedLang of supportedLangs) {
+
+    // locale = langToLocale(supportedLang);
+    const fileName = path.basename(file, '.md');
+    locale = 'af';
+    shell.exec(`mkdir -p ${localeDir}/${locale}/LC_MESSAGES/`);
+    outputFile = path.join(localeDir, locale, 'LC_MESSAGES', `${fileName}.po`);
+    try {
+      fs.statSync(outputFile);
+      // eslint-disable-next-line no-console
+      console.log(`${outputFile} already exists skipping`);
+    } catch (e) {
+      if (e.code === 'ENOENT') {
+        shell.exec(`msginit --no-translator --input=${templateDir}${fileName}.pot
+                    --output-file=${outputFile} -l ${locale}`.replace('\n', ' '));
+      } else {
+        throw e;
+      }
+    }
+
+
+    let po = fs.readFileSync(`${localeDir}/${locale}/LC_MESSAGES/${fileName}.po`, 'utf-8');
+
+    // TODO: fuzzy ok? po2md wants a flag here.
```

Comment:
If this is signalling that fuzzy marked content should be added into the markdown we probably don't want that though we should think about it in the specific context of the markdown translation. 

Usually a fuzzy marked string in the po file means that `msgmerge` has a translated string that fuzzily matches the key. That can be helpful but it is more commonly completely wrong!

On the site we generally don't use fuzzy marked content when building the translations. Instead the translators will see the fuzzy marked content and will fix that translation (sometimes using the fuzzy marked content as a starting point depending on how close it is) - then they remove the fuzzy marker which makes it possible for us to consume it later when we next re-generate the locale js.